lib/services/project-data-service.ts

```diff
@@ -63,7 +63,10 @@ export class ProjectDataService implements IProjectDataService {
 					"description": "NativeScript Application",
 					"license": "SEE LICENSE IN <your-license-filename>",
 					"readme": "NativeScript Application",
-					"repository": "<fill-your-repository-here>"
+					"repository": "<fill-your-repository-here>",
+					"devDependencies": {
```

Comment:
I've pull requests for all three templates. No other templates outside of the organization
NativeScript/template-hello-world-ng#24
NativeScript/template-hello-world#53
NativeScript/template-hello-world-ts#14